index.html

```diff
@@ -3,12 +3,12 @@
   <head>
     <meta charset="utf8"/>
     <title>No Survival - The Game</title>
-    <link href="/styles/layout.css" rel="stylesheet"/>
```

Comment:
Oh, just checked, we changed it because my root is differently, think we talked about it in chat recently